chart/epinio/values.schema.json

```diff
@@ -1,177 +1,380 @@
 {
-  "$schema": "https://json-schema.org/draft-07/schema#",
-  "title": "Values",
-  "type": "object",
-  "properties": {
-    "registryURL": {
-      "type": "string"
-    },
-    "image": {
-      "type": "object",
-      "properties": {
-        "epinio": {
-          "type": "object",
-          "properties": {
-            "repository": {
-              "type": "string"
-            },
-            "tag": {
-              "type": "string"
-            }
-          }
-        },
-        "bash": {
-          "type": "object",
-          "properties": {
-            "repository": {
-              "type": "string"
-            },
-            "tag": {
-              "type": "string"
-            }
-          }
-        },
-        "awscli": {
-          "type": "object",
-          "properties": {
-            "repository": {
-              "type": "string"
-            },
-            "tag": {
-              "type": "string"
-            }
-          }
-        },
-        "kubectl": {
-          "type": "object",
-          "properties": {
-            "repository": {
-              "type": "string"
-            },
-            "tag": {
-              "type": "string"
-            }
-          }
-        }
-      }
-    },
-    "server": {
-      "description": "server configuration",
-      "type": "object",
-      "properties": {
-        "accessControlAllowOrigin": {
-          "type": "string"
-        },
-        "timeoutMultiplier": {
-          "type": "integer"
-        },
-        "sessionKey": {
-          "type": "string"
-        },
-        "traceLevel": {
-          "type": "integer"
-        },
-        "registryCertificateSecret": {
-          "type": "string"
-        },
-        "ingressClassName": {
-          "type": "string"
-        }
-      }
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "title": "Values",
+    "type": "object",
+    "properties": {
+	"registryURL": {
+	    "type": "string"
+	},
+	"image": {
+	    "type": "object",
+	    "properties": {
+		"epinio": {
+		    "type": "object",
+		    "properties": {
```

Comment:
Nothing critical but indentation is broken  :)

Reply:
You saw commit https://github.com/epinio/helm-charts/pull/153/commits/7f67eb1f21712ac99af9cf8251a36ec5297fc1fa ?
What indentation mode was used before that ?

Reply:
Ah, that is an issue with `diff` output (`[+-]`) weirdly interacting with tabs. (The effect helps when looking at YAML, unwanted tabs become more obvious)